examples/worlds/shadow_texture_size.sdf

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" ?>
+<sdf version="1.9">
+  <world name="default">
+    <scene>
+      <background>0.1 0.1 0.1</background>
+      <ambient>0.0 0.0 0.0</ambient>
+    </scene>
+
+    <!--GUI plugins-->
+    <gui>
+      <plugin filename="MinimalScene" name="3D View">
+        <engine>ogre2</engine>
+        <camera_pose>-10 0 7 0 0.5 0</camera_pose>
+        <shadows>
+          <texture_size light_type="directional">16384</texture_size>
```

Comment:
set this to `8192` as I see that's what the tutorial tells the user to do?

Reply:
0d21682c904daa3af7f3c5f860e9eee75f3075f8